setup.py

```diff
@@ -5,14 +5,19 @@
 
 setuptools.setup(
     name="textattack",
-    version="0.0.1",
+    version="0.0.1.7",
     author="QData Lab at the University of Virginia",
     author_email="jm8wx@virginia.edu",
     description="A library for generating text adversarial examples",
+    include_package_data=False,
```

Comment:
just curious, what do this and the changes below do haha

Reply:
This lets you specify specifically which non-.py files should be included along with the code. If it wasn't specified, pip would put the config.json in some other folder, potentially far away from the code. We need the config file to be imported from the code via relative imports, so I changed some settings to allow that